master/internal/kubernetes/pods.go

```diff
@@ -106,8 +103,8 @@ func (p *pods) Receive(ctx *actor.Context) error {
 		p.startPodInformer(ctx)
 		p.startEventListener(ctx)
 
-	case sproto.StartPod:
-		if err := p.receiveStartPod(ctx, msg); err != nil {
+	case sproto.StartTaskPod:
```

Comment:
blocking: can we make `StartTaskPod` and and `KillTaskContainer` messages naming consistent. I would suggest that both RP define their own message type in`sproto` for `KillTaskContainer`. This way task handler call `KillContainer` and then the RP abstracts away that whether it is a pod `KillTaskPod` or a container `KillTaskContainer`. I do realize that both messages have the same exact content which makes it a bit repetitive, but the consistency with how the Start message are names and implemented will improve code readability.